inc/Engine/Media/AboveTheFold/AJAX/Controller.php

```diff
@@ -61,8 +61,9 @@ public function add_data(): array {
 		check_ajax_referer( 'rocket_beacon', 'rocket_beacon_nonce' );
 
 		if ( ! $this->context->is_allowed() ) {
-			wp_send_json_error( 'not allowed' );
-			return; // @phpstan-ignore-line - Needed to prevent further execution.
+			$payload['lcp'] = 'not allowed';
+
+			return $payload;
```

Comment:
AJAX callbacks should return a JSON object, not an array, why do this change?

Reply:
It's from develop, as we discussed, not added by this PR.